download/download.adoc

```diff
@@ -66,7 +66,7 @@ A CR should be almost identical to the Final release.
 * image:download.png[] Download jBPM *#{site.pom.latest.version}* binaries
 ** #{site.pom.latest.jbpmBinZip}[Download jBPM #{site.pom.latest.version}-bin.zip]
 ** #{site.pom.latest.jbpmExamplesZip}[Download jBPM #{site.pom.latest.version}-examples.zip]
-** #{site.pom.latest.jbpmInstallerZip}[Download jBPM #{site.pom.latest.version}-installer.zip]
+** #{site.pom.latest.jbpmInstallerZip}[Download jBPM installer-#{site.pom.latest.version}.zip]
```

Comment:
This looks weird:) What about "jBPM installer 7.0.0.Beta.zip" or even keep the previous name to be consistent with the other entries (I don't think it matters that much that the file name actually changed from `jbpm-<version>installer.zip` to `jbpm-installer-<version>.zip`)